cli/src/vote.rs

```diff
@@ -800,35 +800,49 @@ pub fn process_create_vote_account(
     let fee_payer = config.signers[fee_payer];
     let nonce_authority = config.signers[nonce_authority];
 
+    let is_feature_active = if !sign_only {
+        let feature_address = solana_sdk::feature_set::vote_state_add_vote_latency::id();
+        rpc_client
+            .get_account(&feature_address)
+            .map(|account| {
+                feature::from_account(&account)
+                    .map(|feature| feature.activated_at.is_some())
+                    .unwrap_or(false)
+            })
+            .unwrap_or(false)
+    } else {
+        false
+    };
```

Comment:
```suggestion
    let is_feature_active = sign_only
        .then_some(())
        .and_then(|_| {
            let feature_address =
                solana_sdk::feature_set::vote_state_add_vote_latency::id();
            rpc_client.get_account(&feature_address).ok()
        })
        .and_then(|account| feature::from_account(&account))
        .map_or(false, |feature| feature.activated_at.is_some());
```

i think?

Reply:
this is ofc a nit

Reply:
Ooh, I did think there must be some way to use `bool::then` to make this less awful, but didn't recognize the benefit of using Options throughout. I think your suggestion needs a `not` exclamation point at the beginning, but otherwise looks like it would work. Thx.

Reply:
Pushed. No other changes except for that last commit. One last ✅ , plz?

Reply:
> I think your suggestion needs a not exclamation point at the beginning

ah yeah sry. must've deleted too much while racing the low battery auto shutdown :upside_down_face: